source/date_time.md

```diff
@@ -0,0 +1,89 @@
+# Date and time question types
+
+There are 3 different date and time question types in KoboToolbox: “Date”,
+“Time” and “Date and time”.
+
+The “Date” question type is for capturing date values, for example on a question
+like “What is your date of birth?” In both KoboCollect and Enketo web forms, a
+calendar-style date picker will be shown for selecting the date.
+
+The “Time” question type is for capturing time values, for example on a question
+like “At what time do you leave for work?” In both KoboCollect and Enketo, a
+time picker is shown where a user can select their response.
+
+The third type “Date and time” is for capturing both date and time responses on
+a single question.
+
+## How to set up Date and Time question types
+
+### Setting up in FormBuilder
+
+Adding “Date”, “Time”, and “Date and Time” questions is simple and does not
+require any extra setup:
+
+-   In the KoboToolbox Formbuilder, click the + button to add a new question
+-   Type the question text, for example “What is your date of birth?”, then
+    click ADD QUESTION or press ENTER on your keyboard
+-   Choose the question type
+
+![Adding the questions](images/date_time/adding.gif)
+
+### Setting up in XLSForm
+
+To add date, time, and date and time questions in the XLSForm, use the date,
+time and dateTime question types as shown in the example below:
+
+In XLSForm, you can set up the following:
+
+| type     | name      | label                                         |
+| :------- | :-------- | :-------------------------------------------- |
+| date     | dob       | On what date were you born?                   |
+| time     | time      | At what time do you you leave for work?       |
+| dateTime | date_time | At what date and time did the training start? |
```

Comment:
Let's also include an Excel file with the basic XLSForm used here as we discussed in the last meeting about using both XLSForm directly in the support articles _and_ having something for people to download and use right away.

Reply:
DOne

Reply:
@exprin don't forget to upload the Excel file 🙃

Reply:
The file was uploaded (check the note at the end). I even tested it to check it's downloading

Reply:
@exprin, you need to also include the file in the PR:
<img width="337" alt="Screenshot 2022-06-14 at 10 32 37" src="https://user-images.githubusercontent.com/50016008/173532289-70595f61-d35b-46fa-9638-c26b34f3de81.png">